ARCHITECTURE.md

```
@@ -0,0 +1,155 @@
+| Namespace | Purpose |
+|-----------|---------|
+| `openshift-service-ca-operator` | Operator process runs here |
+| `openshift-service-ca` | Controller Deployment (the operand) runs here |
+| `openshift-config` | User-specified config (read-only) |
+| `openshift-config-managed` | Machine-specified config; CA bundle is synced here |
+
+## Component Overview
+
+Single binary, two processes:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)